README.md

```diff
@@ -5,13 +5,25 @@ Welcome to the pinging pong project, delivered to you by the fabulous ponies !
 This project include a chat with multiple chatroom, and a game interface to see, or play pong parties.
 This is only availaible to 42 students accounts at the moment.
 
-# How to
-Please, include .env file following the diverse .env_sample files present in the project folders. Or else you would get issue while trying to start up the project.
+# How to build the project?
+
+1. Include .env file following the instruction of `.env_sample` files present in the 03 folders (`/backend/.env`, `/frontend/.env`, `./.env`).
+
+2. To buil Dev Environment: `yarn pre-dev`, then `yarn dev`.
+
+3. To build Prod: `yarn build`, then go to the newly created folder `dist_folder` and do `yarn prod`.
+
+Note: In Dev Environment we used NodeJs so we could have hot reloading. In Prod, we used Nginx as the requirement of the project.
```

Comment:
Yes and no, in dev environnement we use vitejs as development server so that we can use react hot reload, which can be use using other tools than vitejs. But in prod, we use nginx, not because of project requirements, but to be closer to real kind of prod web app, I'm not yet an expert on why static website are preferable, but that the reason why we did that  might be good to study why or how to build good prod.

Reply:
Ah ok, thanks for the explaination. I'll change it.